### PR TITLE
Pages and topics

### PR DESCRIPTION
**Pull request by paulinebessoles**
#### :tophat: What? Why?
This PR is related to the documentation update. 

#### :tophat: Done

- [x] Update Pages parent page
- [x] Create Topics page
- [x] Create Pages child page
- [x] Update navigation
- [x] Update screenshots

#### :pushpin: Related Issues
Issues: https://github.com/decidim/documentation/issues/165

#### Testing

:hearts: Thank you!

### CODE REVIEW

File: docs/en/modules/admin/pages/pages/pages.adoc [hunk 1]
```diff
@@ -0,0 +1,82 @@
+= Pages
+
+In this section, you can configure static Pages, to give general information to the participants of the platform.
+
+As an example, some of these pages could refer to:
+
+* What's this platform? Who's the organization?
+* How to participate here?
+* How to create and verify an account?
+* What are the terms of conditions and the legal information about this platform?
```
Comment (carolromero) [inline docs/en/modules/admin/pages/pages/pages.adoc]:
```suggestion
* What are the terms and conditions and the legal information about this platform?
```

File: docs/en/modules/admin/pages/pages/pages.adoc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+== Manage the pages
+
+You can manage the pages from this administration panel:
```
Comment (carolromero) [inline docs/en/modules/admin/pages/pages/pages.adoc]: > 
```suggestion
You can manage the pages from this section of the administration panel: 
```

**Review by carolromero — CHANGES_REQUESTED** [review]
@paulinebessoles just left you a couple of comments for small changes.

**Comment by paulinebessoles** [comment]
@carolromero this PR is ready for new review and merde

**Pull request merged**